src/axom/sidre/core/Group.cpp

```diff
@@ -1790,6 +1790,108 @@ bool Group::isEquivalentTo(const Group* other, bool checkName) const
   return is_equiv;
 }
 
+/*
+ *************************************************************************
+ *
+ * Private class used to set Conduit error handler to default (throw exception)
+ *
+ * Creating an instance of this class will disable the current Conduit error
+ * callbacks.  The default Conduit callbacks throw exceptions when they
+ * encounter I/O errors.  When the instance is destroyed, the previous
+ * callbacks are restored.
+ *
+ * This class is also a functor.  The operator() method takes a closure,
+ * catches any Conduit error, and saves that in the DataStore.
+ *
+ *************************************************************************
+ */
+
+class ConduitErrorSuppressor
+{
+public:
+  using conduit_error_handler = void (*)(const std::string&,
+                                         const std::string&,
+                                         int);
+
+  ConduitErrorSuppressor(const DataStore* ds, bool suppress_in_call = true);
+  ~ConduitErrorSuppressor();
+
+  void operator()(const std::function<void(void)>& conduitOp)
+  {
+    if(m_suppress_in_call)
+    {
+      disable_conduit_error_handlers();
+    }
+
+    try
+    {
+      conduitOp();
+    }
+    catch(conduit::Error& e)
+    {
+      m_ds->setConduitErrorOccurred(true);
+      m_ds->appendToConduitErrors(e.message());
```

Comment:
I might be missing it, but where are the errors cleared? 

Is that the user's responsibility? If so, is it documented and/or exercised in the sidre tests and examples?

Reply:
The user should clear errors as and when the user sees fit.  I'm in the process of adding tests and documentation.